### ISSUE TITLE
Migration to STAC 0.6

### ISSUE THREAD

**Issue by fredliporace** [fredliporace/cbers-2-stac#11]
This issue consolidates all changes that are necessary:

- [ ] Add types to assets ( https://github.com/radiantearth/stac-spec/pull/175, https://github.com/radiantearth/stac-spec/issues/54#issuecomment-413026763  )
- [ ] Remove formats from catalog ( https://github.com/radiantearth/stac-spec/pull/175 )
- [ ] Add version field
- [ ] Array for list of links ( https://github.com/radiantearth/stac-spec/issues/134#issuecomment-413022361, https://github.com/radiantearth/stac-spec/pull/151#issuecomment-413435039 )

**Comment by cholmes** [comment]
Probably should wait for 0.6 to settle down a bit before doing this. Would definitely love you trying out the migration early, to give good feedback before we lock it in, but it feels like a few major things aren't fully settled...

**Comment by fredliporace** [comment]
@cholmes yes, I'm just keeping the work and references organized while receiving the updates from stac repo.
